src/com/walmartlabs/lacinia/pedestal/subscriptions.clj

```diff
@@ -88,19 +88,20 @@
     ;; Keep track of subscriptions by (client-supplied) unique id.
     ;; The value is a shutdown channel that, when closed, triggers
     ;; a cleanup of the subscription.
-    (go-loop [subs {}]
+    (go-loop [subs {}
```

Comment:
It's starting to feel clumsy passing the state around this way; I'd be open to using an explicit Atom (or pair of Atoms) rather than the loop state.

Reply:
Alternately, pass a single map around the loop, pushing subs and connection-payload down a level.